diff --git a/node_modules/react-native-avoid-softinput/android/build.gradle b/node_modules/react-native-avoid-softinput/android/build.gradle
index 8197972..6ac671c 100644
--- a/node_modules/react-native-avoid-softinput/android/build.gradle
+++ b/node_modules/react-native-avoid-softinput/android/build.gradle
@@ -152,7 +152,7 @@ def kotlin_coroutines_version = getExtOrDefault('kotlinCoroutinesVersion')
 def androidX_core_ktx_version = getExtOrDefault('androidXCoreKtxVersion')
 
 dependencies {
-  implementation "com.facebook.react:react-android" // Set by the React Native Gradle Plugin
+  implementation "com.facebook.react:react-native:0.76.6" // Set by the React Native Gradle Plugin
   implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
   implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:$kotlin_coroutines_version"
   implementation "androidx.core:core-ktx:$androidX_core_ktx_version"
